frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1134,30 +1136,49 @@ pub mod pallet {
 	pub enum Event<T: Config> {
 		/// A solution was stored with the given compute.
 		///
-		/// The `origin` indicates the origin of the solution. If `origin` is `Some(AccountId)`,
-		/// the stored solution was submited in the signed phase by a miner with the `AccountId`.
-		/// Otherwise, the solution was stored either during the unsigned phase or by
-		/// `T::ForceOrigin`. The `bool` is `true` when a previous solution was ejected to make
-		/// room for this one.
+		/// If `origin` is `Some(AccountId)`, the stored solution was submitted in the signed phase
+		/// by a miner with the `AccountId`. Otherwise, the solution was stored either during the
+		/// unsigned phase or by `T::ForceOrigin`.
```

Comment:
```suggestion
		/// unsigned phase or by `[T::Config::ForceOrigin]`.
```

I have seen that in all your doc PRs you are not really doing the linking correctly, please be more careful with that.